packages/manager/modules/sharepoint/.htmlhintrc

```diff
@@ -0,0 +1,25 @@
+{
+  "alt-require": true,
+  "attr-lowercase": ["viewBox"],
+  "attr-no-duplication": true,
+  "attr-unsafe-chars": true,
+  "attr-value-double-quotes": true,
+  "attr-value-not-empty": false,
+  "doctype-first": false,
+  "doctype-html5": true,
+  "head-script-disabled": true,
+  "href-abs-or-rel": false,
+  "id-class-ad-disabled": true,
+  "id-class-value": false,
+  "id-unique": false,
+  "inline-script-disabled": false,
+  "inline-style-disabled": false,
+  "space-tab-mixed-disabled": "space",
+  "spec-char-escape": true,
+  "src-not-empty": true,
+  "style-disabled": true,
+  "tag-pair": true,
+  "tag-self-close": false,
+  "tagname-lowercase": true,
+  "title-require": true
+}
```

Comment:
Isn't there a shared configuration for this one ?

Reply:
Not in this case, it's a specific one for that module (options are different from the root file :/)